Enterprise-CMCS/macpro-mdct-mfp
# Adding Export Banner to PDF

### Description

Adding the WP export banner to the PDF and hiding the report header in that page.

### Related ticket(s)


N/A

---
### How to test

- Create a WP
- Review PDF

You should see the following banner:

<img width="1008" alt="Screenshot 2023-12-05 at 2 55 05 PM" src="https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/99458559/fc52a240-51c6-4c5e-b991-534fcd13adbd">

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
